### PR TITLE
Enable nullbility annotations in the project

### PR DESCRIPTION
**Pull request by Numpsy**

### CODE REVIEW

File: src/Serilog.Enrichers.Process/Enrichers/ProcessIdEnricher.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ namespace Serilog.Enrichers
     /// </summary>
     public class ProcessIdEnricher : ILogEventEnricher
     {
-        LogEventProperty _cachedProperty;
+        private LogEventProperty? _cachedProperty;
```
Comment (nblumhardt) [inline src/Serilog.Enrichers.Process/Enrichers/ProcessIdEnricher.cs]: Nitpick; the `serilog/` org projects don't use explicit `private` or `internal`

File: src/Serilog.Enrichers.Process/Enrichers/ProcessNameEnricher.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ namespace Serilog.Enrichers
     /// </summary>
     public class ProcessNameEnricher : ILogEventEnricher
     {
-        LogEventProperty _cachedProperty;
+        private LogEventProperty? _cachedProperty;
```
Comment (nblumhardt) [inline src/Serilog.Enrichers.Process/Enrichers/ProcessNameEnricher.cs]: As above

**Review by nblumhardt — APPROVED** [review]
LGTM, just nitpicky coding style feedback :-)

File: src/Serilog.Enrichers.Process/Enrichers/ProcessIdEnricher.cs (hunk 1 above)
Comment (Numpsy) [reply src/Serilog.Enrichers.Process/Enrichers/ProcessIdEnricher.cs]: Ok.
Actually, that makes me wonder if we should import the .editorconfig out of the main repo to help make the settings consistent (the one here is basically empty)

**Comment by Numpsy** [comment]
@nblumhardt Whilst you're in the area - looks like the nuget publishing key in the CI build needs updating

**Comment by nblumhardt** [comment]
@Numpsy on it now, thanks! :-)

**Pull request merged**